source/SSBGameScene.h

```diff
@@ -75,6 +75,8 @@ class GameScene : public scene2::Scene2 {
     std::shared_ptr<scene2::PolygonNode> _leftnode;
     /** Reference to the right joystick image */
     std::shared_ptr<scene2::PolygonNode> _rightnode;
+    /** The camera for this scene */
+    std::shared_ptr<scene2::ScrollPane> scroll_pane;
```

Comment:
```suggestion
    std::shared_ptr<scene2::ScrollPane> _scrollPane;
```
could you change the variable to _scrollPane to keep it consistent